spec/gruf/newrelic/server_interceptor_spec.rb

```diff
@@ -43,12 +43,22 @@
     it 'should trace the request' do
       expect(interceptor).to receive(:perform_action_with_newrelic_trace).with(
         category: Gruf::Newrelic.server_category,
-        class_name: request.service,
+        class_name: request.service.name,
```

Comment:
I believe `.name` is not needed here, since for a Class instance it is the same effectively as `.to_s`

Reply:
We are coercing the class to a string with the `join` in the server_interceptor. So the result will be a stringified class name, where as this is expecting the class itself. Hence, we expect the stringified class name